### DISCUSSION TITLE
DPS Defensives

### DISCUSSION THREAD

**Discussion by ColonialAF** [Hekili/hekili#932] [Ideas]
Hello!

Searching about the Internet, it looks like, at one point, defensives for non-tank specs was enabled, but in the current vrsion, the mouseoevr text still says it only works for tank specs, and that does seem to be the case. For my unholy DK especially, I would like the defensives to pop on. Also would be useful for my other DPS specs (like Arms Warrior), but it was with my DK that I really noticed it.

Is there a way to re-enale that for non-tanks?

**Comment by Hekili** [comment]
It's not that they're disabled, it's that using your defensive abilities aren't part of your sim priority.  You can write a custom priority that includes them, if you wish.

https://github.com/Hekili/hekili/wiki/Frequently-Asked-Questions

**Reply by ColonialAF** [reply]
That makes sense, but it's also why I like to keep then "separate." They're obviously far more situational, but a lot of times I need the reminder htat they're off cooldown or that I can use them.

**Comment by Hekili** [comment]
That's totally valid, it's just a matter of personal preference **or** raid strategy in terms of how you use them, usually.  I don't like the addon to recommend stuff that is probably raid strategy.  That's why tanks mainly get recommendations related to maintenance buffs, or major defensives are recommended AFTER you take a big hit and you might die if you don't do something about it.

**Reply by ColonialAF** [reply]
Alright, I've started working on my own action lists (using the defaults as a base, then copying). Is there a way to have certain abilities only trigger if I'm wearing certain equipment?

For example, when I'm specced for Protection, and wearing The Wall, I want to set Shield Wall to pop up more often. When I'm using Thunderlord, I don't.

**Reply by Hekili** [reply]
Generally, yes -- the same way you would in SimulationCraft.  You can reference their wiki for a lot of guidance related to writing APLs.

**Reply by ColonialAF** [reply]
All I can find there is howt o set it up for their sims.

I tried this:
shoulders,name=the_wall

But that always returns "true" because all those values are, apparently, nil

**Reply by Hekili** [reply]
You'll have to refer to their wiki.  I do not help write custom priorities.

**Reply by ColonialAF** [reply]
Yeah that's where I was, but I understand. Thanks for your help to here, at least. :)
